core/dbt/task/runnable.py

```diff
@@ -461,7 +459,9 @@ def run(self):
             )
 
         if len(self._flattened_nodes) == 0:
-            warn_or_error("\nWARNING: Nothing to do. Try checking your model "
+            with TextOnly():
+                fire_event(EmptyLine())
+            warn_or_error("WARNING: Nothing to do. Try checking your model "
```

Comment:
Take out the "WARNING" text too since we want to remove that duplication.

Reply:
We're no longer including the log level in the CLI output (since https://github.com/dbt-labs/dbt-core/pull/4379), so it's not as duplicative as it used to be. But I can sub this out to use [`ui.warning_tag`](https://github.com/dbt-labs/dbt-core/blob/29f504e201ba512c30f59de648a00c54ac24863f/core/dbt/ui.py#L68)

Reply:
I realize I changed `warning_tag` to do nothing! Let me change it to do something again

Reply:
I'm thinking we could use these for deprecation warnings too, just to give them a little colorful pop like the others:
<img width="959" alt="Screenshot 2021-12-02 at 18 20 05" src="https://user-images.githubusercontent.com/13897643/144471402-0f82a6ad-4d6d-4d2d-9b2f-6ff11a24ce9d.png">

Reply:
@emmyoop let me know if you hate it!